src/services/__tests__/QueueService.int.spec.ts

```diff
@@ -0,0 +1,112 @@
+import { joinQueue } from "../QueueService";
+import { leaveQueue } from "../QueueService";
+import { BallChaserQueueBuilder, LeaderboardBuilder } from "../../../.jest/Builder";
+import { DateTime } from "luxon";
+import getEnvVariable from "../../utils/getEnvVariable";
+import { PrismaClient } from "@prisma/client";
+import LeaderboardRepository from "../../repositories/LeaderboardRepository";
+import { mocked } from "ts-jest/utils";
+import { join } from "path/posix";
```

Comment:
Are we using this?

Reply:
Good catch. It was from a typo earlier that immediately imported it.